mdata/aggmetrics.go

```diff
@@ -49,8 +44,8 @@ func (ms *AggMetrics) GC() {
 		time.Sleep(diff + time.Minute)
 		log.Info("checking for stale chunks that need persisting.")
 		now := uint32(time.Now().Unix())
-		chunkMinTs := now - (now % ms.chunkSpan) - uint32(ms.chunkMaxStale)
-		metricMinTs := now - (now % ms.chunkSpan) - uint32(ms.metricMaxStale)
+		chunkMinTs := now - uint32(ms.chunkMaxStale)
+		metricMinTs := now - uint32(ms.metricMaxStale)
```

Comment:
I don't recall why we had to subtract the chunkspan for these calculations.  if we still have to do this, this would be trickier, as the chunkspan is specific to the metric that we'll check.